Source/Scene/Scene.js

```diff
@@ -2922,6 +2968,18 @@ define([
         context.endFrame();
     }
 
+    function tryAndCatchError(scene, time, functionToExecute) {
+        try {
+            functionToExecute(scene, time);
+        } catch (error) {
+            scene._renderError.raiseEvent(scene, error);
```

Comment:
Both `update` and `render` use `renderError`. Do we need an `updateError`?

Reply:
The only place it's used in Cesium is that the viewer listens to it, and stops the render loop and displays it, which we'd still want. In case any app is also listening for this event, errors occurring during `update` would no longer be caught, which might be an issue.

Both are occurring during the `scene.render` function, so I don't think the naming is an issue, but if we wanted we could rename and deprecate `scene.renderError`

Reply:
Probably okay to leave as is for now. Maybe we can reevaluate for the upcoming PR.